test/e2e/gui/elements/check_box.py

```diff
@@ -14,7 +14,7 @@ class CheckBox(QObject):
     @allure.step("Set {0} value: {1}")
     def set(self, value: bool):
         if self.is_checked is not value:
-            self.click()
+            self.click(x=5, y=5)
```

Comment:
wouldn't self.click(x=width * 0.5, y=height * 0.5) be safer? Not sure what x=5;y=5 would mean for every control.

Reply:
it is not for every control. it is only for checkbox that will be clicked with (width=5, height=5), it is done specifically to click checkbox on onboarding screen where checkbox has Terms and Conditions link. So when i click that checkbox in the middle, it clicks the link instead @alexjba

Reply:
i will do a click for indicator instead for checkboxes , thanks for hints :)

```
--- ui/StatusQ/src/StatusQ/Controls/StatusCheckBox.qml
+++ ui/StatusQ/src/StatusQ/Controls/StatusCheckBox.qml
@@ -43,6 +43,7 @@ CheckBox {
     font.pixelSize: size === StatusCheckBox.Size.Regular ? 15 : 13
 
     indicator: Rectangle {
+        objectName: "indicator"
         anchors.left: root.leftSide? parent.left : undefined
         anchors.right: !root.leftSide? parent.right : undefined
         implicitWidth: size === StatusCheckBox.Size.Regular
```